server/service/service_test.go

```diff
@@ -0,0 +1,128 @@
+package service
```

Comment:
Please add a license header to all new files. I've just submitted a PR that adds them to all the other files so just look at one of them for reference.

Reply:
sure, I will

Reply:
Also note that packages must be preceded with a one line comment explaining the purpose of the package